lib/vector-core/src/metrics/label_filter.rs

```diff
@@ -1,11 +1,11 @@
-use metrics::Label;
+use metrics::{KeyName, Label};
 use metrics_tracing_context::LabelFilter;
 
 #[derive(Debug, Clone)]
 pub(crate) struct VectorLabelFilter;
 
 impl LabelFilter for VectorLabelFilter {
-    fn should_include_label(&self, label: &Label) -> bool {
+    fn should_include_label(&self, _key: &KeyName, label: &Label) -> bool {
         let key = label.key();
```

Comment:
Should this be using the `key` that is passed in now? If it is available on `Label` it does seem odd that they added it to the signature here 🤔

Reply:
I opted to follow what the existing `impl`s in the crate updated to: https://github.com/metrics-rs/metrics/blob/main/metrics-tracing-context/src/label_filter.rs#L43-L47

Reply:
Ah, nvm, the `KeyName` here appears to be the metric name. Maybe we could call it `_metric_name` to make that clearer?